runbotics/runbotics-orchestrator-ui/src/src-app/components/cron/utils.ts

```diff
@@ -71,3 +71,6 @@ export function usePrevious(value) {
 
     return ref.current;
 }
+
+export const checkIsArray = (shortcuts: any): shortcuts is ShortcutsType => Array.isArray(shortcuts);
```

Comment:
here is still smthing wrong.
1. parameter should be `unknown` instead of `any`
2. `ShortcutsType` is a string union so its strange that you check if its array. It should be a single value looking at the return type
3. I suppose that you want to check if the elements of the array are included in `ShortcutsType` so the return type should be `ShortcutsType[]`

maybe we can discuss it on a call

P.S. imo the interface should be `ShortcutType`

Reply:
1. done  P.S. done

Reply:
solved